marian13/convenient_service
# CS-Y: Add a new logo

## What was done as a part of this PR?


- Replaced placeholder logo.

## Why it was done?



- Each great idea should have its own unique visual representation.

## How it was done?


- Done with love, done by hand.

## How to test?

- Change the GitHub theme in settings to check image adaptation.
![image](https://user-images.githubusercontent.com/55796418/202819641-cd003d02-a290-41c1-9476-2000665be99e.png)
![image](https://user-images.githubusercontent.com/55796418/202819757-b5a324ba-f301-45e6-9e3f-69dcd737e247.png)
